itests/org.jboss.tools.quarkus.integration.tests/src/org/jboss/tools/quarkus/integration/tests/project/CreateNewProjectTest.java

```diff
@@ -27,36 +36,73 @@
 /**
  * 
  * @author jkopriva@redhat.com
+ * @editor olkornii@redhat.com
  */
 @OpenPerspective(QuarkusPerspective.class)
 @RunWith(RedDeerSuite.class)
 public class CreateNewProjectTest {
 
 	private static String MAVEN_PROJECT_NAME = "testProjectMaven";
-
+	private static String GRADLE_PROJECT_NAME = "testProjectGradle"; 
+	
 	@Test
 	public void testNewNewQuarkusMavenProject() {
 		new WorkbenchShell().setFocus();
 
-		QuarkusWizard qw = new QuarkusWizard();
-		qw.open();
-		assertTrue(qw.isOpen());
+		QuarkusWizard qw = new QuarkusWizard(); // create object QuarkusWizard
```

Comment:
@olkornii can you please delete all these "debug comments"? It makes the changes in code harder to see.